lib/geordi/dump_loader.rb

```diff
@@ -61,9 +80,12 @@ def dump_file
     end
 
     def load
+      adapter_command = "#{config['adapter']}_command"
+      Interaction.fail "Failure to parse config/database.yml. Unknown database adapter #{config['adapter'].inspect}." unless defined? adapter_command
```

Comment:
Didn't parsing actually succeed? What about
```suggestion
      Interaction.fail "Unknown database adapter #{config['adapter'].inspect} in config/database.yml." unless defined? adapter_command
```